[2023-10-26T07:08:42.835+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T06:58:30.719052+00:00 [queued]>
[2023-10-26T07:08:42.856+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T06:58:30.719052+00:00 [queued]>
[2023-10-26T07:08:42.857+0000] {taskinstance.py:1359} INFO - Starting attempt 3 of 3
[2023-10-26T07:08:42.878+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2023-10-26 06:58:30.719052+00:00
[2023-10-26T07:08:42.892+0000] {standard_task_runner.py:57} INFO - Started process 159 to run task
[2023-10-26T07:08:42.895+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_data', 'manual__2023-10-26T06:58:30.719052+00:00', '--job-id', '464', '--raw', '--subdir', 'DAGS_FOLDER/tugaskel8_dag.py', '--cfg-path', '/tmp/tmpn0skcc21']
[2023-10-26T07:08:42.899+0000] {standard_task_runner.py:85} INFO - Job 464: Subtask load_data
[2023-10-26T07:08:42.968+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_dag.load_data manual__2023-10-26T06:58:30.719052+00:00 [running]> on host d73c1ff20372
[2023-10-26T07:08:43.299+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel8' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T06:58:30.719052+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-26T06:58:30.719052+00:00'
[2023-10-26T07:08:43.390+0000] {base.py:73} INFO - Using connection ID 'db_polusiudara' for task execution.
[2023-10-26T07:08:45.100+0000] {python.py:194} INFO - Done. Returned value was: Sukses Memasukkan Data
[2023-10-26T07:08:45.237+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_data, execution_date=20231026T065830, start_date=20231026T070842, end_date=20231026T070845
[2023-10-26T07:08:45.282+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-26T07:08:45.305+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
